### ISSUE TITLE
Make compatible with Rollup 1.0.0

### ISSUE THREAD

**Issue by jonnitto** [ko-yelie/gulp-rollup-each#14]
The current version of gulp-rollup-each doesn't work with [Rollup 1.0.0](https://github.com/rollup/rollup/releases/tag/v1.0.0)

**Comment by panoply** [comment]
@jonnitto You should import the rollup object by its dependency. Don't use `require('rollup')` like the docs. Instead use `import` and include the rollup object, eg:

This option of importing will work with v1.0.0.

```
import rollupEach from 'gulp-rollup-each'
import rollup from 'rollup'

rollupEach({  },{ }, rollup)

```

This option of importing will **not** work with 1.0.0.

```
import rollupEach from 'gulp-rollup-each'

rollupEach({  },{ }, require('rollup'))
```

**Comment by jonnitto** [comment]
Thanks for the fast answer. But node.js doesn't support import statements yet (Or at least it is not enabled by default). https://nodejs.org/api/esm.html#esm_enabling As I use rollup for a published build-pipeline, I can't use experimental features…

**Comment by panoply** [comment]
Have you tried `require('rollup').rollup`?

**Comment by jonnitto** [comment]
**First of all, thank you very much for your helping hand. This is awesome!**

Yep. I tried `.rollup` But this makes no sense, as the script expects the rollup object: https://github.com/ko-yelie/gulp-rollup-each/blob/master/lib/index.js#L12

If I pass the rollup object, I still get the error:
```js
UnhandledPromiseRejectionWarning: TypeError [ERR_INVALID_ARG_TYPE]: The first argument must be one of type string, Buffer, ArrayBuffer, Array, or Array-like Object. Received type undefined
    at Function.from (buffer.js:200:11)
    at rollup.then.then.result (node_modules/gulp-rollup-each/lib/index.js:50:34)
```

**Comment by panoply** [comment]
@jonnitto I have a PR #16  which fixes this issue and supports Rollup v1.0.0.  @ko-yelie will just need to update NPM package version and merge it.

**Comment by ko-yelie** [comment]
I have released gulp-rollup-each v3.0.0 which supports Rollup v1.0.0.
